packages/2d/src/curves/CurveSampler.ts

```diff
@@ -0,0 +1,28 @@
+import {Vector2} from '@motion-canvas/core/lib/types';
+
+export interface SampledPoint {
+  t: number;
+  position: Vector2;
+  distance: number;
+  tangent: Vector2;
+  previous?: SampledPoint;
+  next?: SampledPoint;
+}
+
+export interface CurveSampler {
```

Comment:
Interfaces for classes should be used only when there are two or more actual classes implementing them.
The fact that the only class implementing this is called `Uniform` makes me think that this interface is not necessary.

Reply:
Fair enough. I removed the interface.